scripts/index.js

```diff
@@ -0,0 +1,162 @@
+const lookupTable = {
+  110: ["#321", "☀️🔥☀️🐜🍂🔥🍂🦂🔥"],
+  100: ["#731", "☀️🍂🌵🥀🦂🌵🍂🐜🍂"],
+  90: ["darkred", "🌵🦎🍂🌴🐍🌵🦂🌵🥀"],
+  80: ["red", "🏜🌴🌵🐍🌻🦂🌵🌺🌵🐍"],
+  70: ["orange", "🌸🌿🌼🌷🌻🌿☘️🌱🌻🌷"],
+  60: ["yellow", "🌾🌾🍃🌱🛤🌾☘️🌾🌾🍃"],
+  50: ["#1b5", "🌲🍃🍄🌲🍁🌲🌲🍂🌲"],
+  40: ["#00b8b8", "🌧🌲💨🍄🌲🌧🌲🍂🌲🌧"],
+  32: ["#2de", "🌧💨⛄️🌲🌧⛄️🌧💨🌲🌧"],
+  22: ["#29f", "❄️⛄️💨🌧⛄️❄️💨🌲💨🌨⛄️❄️"],
+  21: ["#13d", "❄️💨🌨❄️💨❄️☃️❄️💨🌨"],
+  sunny: "☁️ ☁️ ☁️ ☀️ ☁️ ☁️",
+  cloudy: "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️",
+  rainy: "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧",
+  snowy: "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨",
+  windy: "🌬💨🌬💨🌬💨🌬💨",
+  foggy: "🌫☁️🌫☁️🌫☁️🌫☁️🌫",
+};
+const state = {
+  city: "Berkeley",
+  temperature: 100,
+  unit: "fahrenheit",
+  sky: "sunny",
+};
+const setTemp = (event) => {
+  if (event) {
+    newVal = event.target.value
+      ? parseInt(event.target.value)
+      : parseInt(event.target.dataset.temp);
+
+    state.temperature += newVal;
+  }
+  updateUI();
+};
+
+const updateUI = () => {
+  const temperature = state.unit === "fahrenheit" ? state.temperature : Math.round((state.temperature * 9/5 + 32));
+
+  let currentColorCode;
+  let gardenContents;
+
+  const temperatureSpan = document.getElementById("temp-today");
+  const gardenPlot = document.getElementById("garden-plot");
+
+  if (temperature < 22) {
+    currentColorCode = lookupTable[21][0];
+    gardenContents = lookupTable[21][1];
```

Comment:
great way to use the conditional for currentColorCode and gardenContents at once.